File: server/EAccountingServer.Application/Models/Dtos/CompanyUsers/CompanyUserListDto.cs
using EAccountingServer.Application.Models.Dtos.Companies;

namespace EAccountingServer.Application.Models.Dtos.CompanyUsers
{
    public class CompanyUserListDto
    {
        public Guid CompanyId { get; set; }
        public CompanyListDto Company { get; set; }
        public Guid UserId { get; set; }
    }
}

File: client/src/models/CompanyUsers/CompanyUserListDto.ts
import type { CompanyListDto } from "@/models/Companies/CompanyListDto"

export class CompanyUserListDto{
    companyId: string
    company: CompanyListDto
    userId: string
    
    constructor(companyId: any, company: CompanyListDto, userId: string) {
        this.companyId = companyId
        this.company = company
        this.userId = userId
    }
}